packages/editor-ui/src/components/executions/workflow/WorkflowExecutionsList.vue

```diff
@@ -120,6 +120,8 @@ onBeforeRouteLeave(async (to, _, next) => {
 </template>
 
 <style module lang="scss">
+@use 'n8n-design-system/css/mixins/breakpoints' as mixins;
```

Comment:
Instead of adding this mixin to all the components, couldn't we just import it once in `packages/design-system/src/css/index.scss`?

Reply:
Imported all mixins globally via `vite.config.ts`